### DISCUSSION TITLE
跑faster rcnn，我已经修改了配置文件中的类的数量，训练时仍会报和COCO 的80类不匹配

### DISCUSSION THREAD

**Discussion by FengRongYue** [open-mmlab/mmdetection#9813] [Q&A]
报错
![814b6d1ae207cd85fec9e86169994dc](https://user-images.githubusercontent.com/95663126/220307734-b196a97b-f795-498d-ae39-960192f01ff6.jpg)
我的配置文件
![288b08cd1d466e0d1a913c10728afba](https://user-images.githubusercontent.com/95663126/220307860-8e3e5628-0c05-46e8-a36e-e64f4472fbeb.jpg)
网上的解决办法都是改源码什么的，感觉完全不靠谱啊，我觉得一定是我配置文件写的不对，但是不知道哪里不对。fasterrcnn的配置文件里面我只找到了一个地方该类的数量的。

**Comment by Renzhihan** [comment]
dataset的config文件中也需要改一下类别名。

**Comment by initiater** [comment]
改完model中的类别数 还有mmdet下的coco.py 以及core下的evaluation中的class_name.py中的类别数量
最后别忘了 setup.py install   如果不行 直接去源码里面修改
